Disable the installation of the bash completion, as it attempts to put it in /usr/
--- src/CMakeLists.txt.orig	2022-07-05 20:22:56.563545000 +0100
+++ src/CMakeLists.txt	2022-07-05 20:23:03.887567311 +0100
@@ -232,10 +232,6 @@
   set (completiondir "/usr/local/etc/bash_completion.d")
 endif()
 
-install(FILES contrib/bashcomp/neper-completion.bash
-        DESTINATION ${completiondir}
-        RENAME neper)
-
 # Testing ######################################################################
 
 include(CTest)
